specs-go/config.go

```diff
@@ -112,12 +112,11 @@ type Platform struct {
 
 // Mount specifies a mount for a container.
 type Mount struct {
-	// Destination is the path where the mount will be placed relative to the container's root.  The path and child directories MUST exist, a runtime MUST NOT create directories automatically to a mount point.
+	// Destination is the absolute path where the mount will be placed in the container.
 	Destination string `json:"destination"`
 	// Type specifies the mount kind.
-	Type string `json:"type,omitempty"`
-	// Source specifies the source path of the mount.  In the case of bind mounts on
-	// Linux based systems this would be the file on the host.
+	Type string `json:"type,omitempty" platform:"linux,solaris"`
```

Comment:
Currently `type` is still *defined* for Windows, we just [require it to be unset][1].  I'd rather move it to a POSIX-specific subsection ([like we already do for UID/GID][2]) so the field would be undefined in the Windows-track spec.  More on this distinction [here][3] and in #680.  But as long as the Markdown docs define the property for Windows (even if they forbid it), I don't think we want to add this `platform` tag here.

[1]: https://github.com/opencontainers/runtime-spec/blame/2ec18c02474dd49ce20aa30236959a8a80763042/config.md#L66
[2]: https://github.com/opencontainers/runtime-spec/blob/2ec18c02474dd49ce20aa30236959a8a80763042/config.md#linux-and-solaris-user
[3]: https://github.com/opencontainers/runtime-spec/pull/747/files#diff-958e7270f96f5407d7d980f500805b1bR58